src/main/java/race/CarRace.java

```diff
@@ -0,0 +1,91 @@
+package race;
+
+public class CarRace {
+
+    private int carCount;
+
+    private int gameCount;
+
+    private String[] carDistance;
+
+    private final String messageCarCount = "자동차 대수는 몇 대 인가요?";
+
+    private final String messageGameCount = "시도할 회수는 몇 회 인가요?";
```

Comment:
```suggestion
    private static final String messageCarCount = "자동차 대수는 몇 대 인가요?";
    private static final String messageGameCount = "시도할 회수는 몇 회 인가요?";
    
    private int carCount;
    private int gameCount;
    private String[] carDistance;
```
이런식으로 상수와 멤버변수를 구분해주면 어떨까요?